js/kanji.js

```diff
@@ -150,6 +151,52 @@ window.Kanji =  {
     }
   },
 
+  _initialSelectAllHide: function() {
+    $(".category-select-all").each(function() {
+      $(this).hide();
+    });
+    $(".category-deselect-all").each(function() {
+      $(this).hide();
+    });
+    $(".category-select-all-show").each(function() {
+      $(this).show();
+    });
+  },
+
+  __selectClicker: function() {
+    if (!$(this).hasClass('selected')) {
+      $(this).trigger("click");
+    }
+  },
+
+  __deselectClicker: function() {
+    if ($(this).hasClass('selected')) {
+      $(this).trigger("click");
```

Comment:
Same thing about single quotes here, and some other places in the new additions. :+1: